apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -292,16 +305,22 @@ export const ModerationNotification = (props) => {
               <p>{commentText}</p>
             </div>
           </div>
-          <ModerationNotificationActionsBar
-            isPending={notification.has_pending_notifications}
-            isDisabled={notification.moderator_statement}
-            isBlocked={notification.is_blocked}
-            isHighlighted={notification.is_moderator_marked}
-            onToggleForm={() => toggleModerationStatementForm()}
-            onToggleBlock={() => toggleIsBlocked()}
-            onToggleHighlight={() => toggleIsHighlighted()}
-            onTogglePending={() => toggleIsPending()}
-          />
+          {loading
+            ? (
+              <div className="d-flex justify-content-center">
+                <i className="fa fa-spinner fa-pulse" aria-hidden="true" />
+              </div>)
```

Comment:
this shrinks the notification and looks broken when loading is fast, would add some height somehow so it doesn't look like the notification collapses